app/views/answers/_vote_answer.html.erb

```diff
@@ -0,0 +1,7 @@
+Vote Total:<%=@answer.total_votes%>
+
+<% if logged_in? && current_user != @answer.user && current_user.has_voted?(@answer) == false%>
```

Comment:
I might do `if @answer.votable_by(current_ser)` and do the eligibility checks in the model.